lib/fiware-iotagent-lib.js

```diff
@@ -84,6 +84,16 @@ function doActivate(newConfig, callback) {
         commandRegistry,
         securityService;
 
+    logger.format = logger.formatters.pipe;
+
+    config.setConfig(newConfig); //moving up here because otherwise env variable are not considered by the code below
+
+    if (!config.getConfig().dieOnUnexpectedError) {
+        process.on('uncaughtException', globalErrorHandler);
+    }
+
```

Comment:
@AlvaroVega not removed, but moved upfront ;)

Reply:
i moved upfront in the code, to be sure that env variables are all checked before